content/pl/docs/installation/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Instalacja systemu
 linkTitle: Instalacja systemu
-weight: 3
+weight: 4
```

Comment:
Maksymalna głębia jaka powinna być to 3. Wszystko głębiej będzie już nieczytelne.